rruleset.go

```diff
@@ -101,6 +105,10 @@ func (set *Set) ExDate(exdate time.Time) {
 	set.exdate = append(set.exdate, exdate)
 }
 
+func (set *Set) SetExDate(exdates []time.Time) {
```

Comment:
SetExDates might be a better name?
Also, since this is exported can we include a comment description? Same with the other function.

Reply:
- Changed both function names from singular to plural
- Added comment descriptions to both functions